contrib/test.sh

```diff
@@ -49,3 +49,16 @@ if [ "$DO_BENCH" = true ]
 then
     cargo bench --features="unstable compiler"
 fi
+
+# Run Integration tests if told so
+if [ -n "$BITCOINVERSION" ]; then
+    set -e
+    cd integration_test  
+    curl https://bitcoincore.org/bin/bitcoin-core-$BITCOINVERSION/bitcoin-$BITCOINVERSION-x86_64-linux-gnu.tar.gz | tar xvzf - bitcoin-$BITCOINVERSION/bin/bitcoind    # will abort if the check fails.
+    sha256sum --check bitcoin-core-$BITCOINVERSION.sha256sum
```

Comment:
I think you're forgetting to download the sha256sum? At least,when I try running this with 0.21.1 or 0.20.1 it doesn't work here.

Reply:
The file would have to be manually downloaded, sha256sum checked locally, and then finally committed here. Unfornutealy, this will be a manual process for every release of bitcoin core. 

I don't see the point of downloading the sha2 sum from bitcoincore.org if we are downloading the binary from there :P .

Reply:
Ah, perfect, I understand. I don't know how I didn't see the sha256sum file checked in for 0.20.0 (now 0.21.0).

Reply:
Will update the version to 0.21.0